src/instructlab/model/chat.py

```diff
@@ -175,7 +201,19 @@ def chat(
 
     # TODO: this whole code block is replicated in generate.py. Refactor to a common function.
```

Comment:
This comment is suspicious. Should we also update `generate` code to do the same - resolve the local endpoint if it's running?

Reply:
The comment was just copied over from an earlier section in the code. I believe the issue for `generate` should be partially solved in PR https://github.com/instructlab/instructlab/pull/1901. We should keep the scope of this PR to just chat for now.

Reply:
Ignore my previous comment... this isn't even a line that gets touched in this PR. Lets please limit the review comments to what is being changed.

Reply:
It's not about particular lines. At the very least, this PR affects the comment since now it's invalid (because the PR modifies the code - it's no longer correct that the block replicates anything. It doesn't anymore.)

You can remove the comment (since it's invalid), or modify the `generate.py` to keep the comment valid.

Reply:
That's a very good point, it sounds like the best way forward will be to remove the comment.